lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.PlantLoop.rb

```diff
@@ -91,32 +91,70 @@ def plant_loop_apply_prm_number_of_cooling_towers(plant_loop)
   end
 
   # Splits the single chiller used for the initial sizing run
-  # into multiple separate chillers based on Appendix G.
-  #
+  # into multiple separate chillers based on Appendix G. Also applies
+  # EMS to stage chillers properly
   # @param plant_loop [OpenStudio::Model::PlantLoop] chilled water loop
-  # @param sizing_run_dir [String] sizing run directory
   # @return [Boolean] returns true if successful, false if not
-  def plant_loop_apply_prm_number_of_chillers(plant_loop, sizing_run_dir = nil)
+  def plant_loop_apply_prm_number_of_chillers(model, plant_loop)
```

Comment:
I commenting here because I'm seeing this change but I think that it applies to other methods in that same file.

I don't think we need `model` as an argument because this file is about plant loop methods. Also, `model` can be obtained if needed by `plant_loop.model`.

Reply:
If we do want to keep `model` as an argument, then it is missing from the YARD docs.

Reply:
Good point! I'll remove it.